src/main/java/com.company/linkedlist/ReverseLinkedList.java

```diff
@@ -0,0 +1,14 @@
+package com.company.linkedlist;
+
+public class ReverseLinkedList {
+    ListNode reverseLinkedList(ListNode head) {
+        ListNode current = head;
+        ListNode prev = null;
+        while(current != null) {
+            current.next = prev;
+            prev = current;
+            current = current.next;
```

Comment:
You already broke the link for current. How will you able move to next node?

Did you run it with an example?

Reply:
fixed and checked in leetcode